src/Migrator.sol

```diff
@@ -13,68 +12,58 @@ import "src/interfaces/balancer/IAsset.sol";
 import "src/interfaces/balancer/IManagedPool.sol";
 import "src/interfaces/balancer/WeightedMath.sol";
 import "src/interfaces/aura/IRewardPool4626.sol";
-import "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
-import "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import "solmate/src/tokens/ERC20.sol";
+import "solmate/src/tokens/WETH.sol";
+import "solmate/src/utils/SafeTransferLib.sol";
 import "openzeppelin-contracts/contracts/access/Ownable.sol";
-import "openzeppelin-contracts/contracts/proxy/utils/Initializable.sol";
-import "openzeppelin-contracts/contracts/proxy/transparent/ProxyAdmin.sol";
-import "openzeppelin-contracts/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
 
 /**
- * @title  Sushi to Balancer migrator
- * @notice Tool to facilitate migrating a sushi SLP position to a BPT or auraBPT position
+ * @title  Sushi to Balancer LP migrator
+ * @notice Tool to facilitate migrating Sushi LPs to Balancer or Aura
+ * @dev SLP: SushiSwap LP Token
+ * @dev BPT: 20WETH-80TOKEN Balancer Pool Token
+ * @dev auraBPT: 20WETH-80TOKEN Aura Deposit Vault
  */
-contract Migrator is IMigrator, Initializable, Ownable {
-    using SafeERC20 for IERC20;
+contract Migrator is IMigrator, Ownable {
+    using SafeTransferLib for ERC20;
 
-    uint256 public constant BPS = 10000;
-    uint256 public unrwapSlippage = 10; // 0.1%
     uint256 public swapSlippage = 10; // 0.1%
 
-    bytes32 public balancerPoolId;
-
-    IWETH9 public weth;
-    IERC20 public token;
-    IERC20 public bpt;
-    IERC20 public auraBpt;
-    IUniswapV2Pair public slp;
-    IRewardPool4626 public auraPool;
-    AggregatorV3Interface public priceFeed;
-    IUniswapV2Router02 public sushiRouter;
-    IVault public balancerVault;
-    IBasePool public balancerPool;
-    IAsset[] public poolAssets = new IAsset[](2);
+    uint256 public immutable BPS = 10000;
```

Comment:
note: This works but `constant` is fine too.

Reply:
changed to immutable to have more consistency